docs/_writing-tests/reftests.md

```diff
@@ -112,10 +112,47 @@ only guaranteed to be after those events.
 ## Fuzzy Matching
 
 In some situations a test may have subtle differences in rendering
-compared to the reference due to, e.g., anti-aliasing. This may cause
-the test to pass on some platforms but fail on others. In this case
-some affordance for subtle discrepancies is desirable. However no
-mechanism to allow this has yet been standardized.
+compared to the reference due to, e.g., anti-aliasing. To allow for
+these small differences, we allow tests to specify a fuzziness
+characterised by two parameters:
+
+ * A maximum difference in the per-channel color value for any pixel.
+ * A maximum number of total pixels that may be different
+
+To specify the fuzziness in the test file one may add a `<meta
+name=fuzzy>` element (or, in the case of more complex tests, to any
+page containing the `<href rel=[mis]match>` elements). In the simplest
```

Comment:
So if there are multiple `<link rel=match>` elements, what happens? Is the fuzziness applied to all alternations? That's another change from Gecko, right?

(also `<link`, not `<href`)

Reply:
Oh this is covered below.